src/test/java/crawlercommons/sitemaps/SiteMapParserExtensionTest.java

```diff
@@ -74,12 +78,25 @@ public void testVideosSitemap() throws UnknownFormatException, IOException {
         expectedVideoAttributes.setUploader("GrillyMcGrillerson");
         expectedVideoAttributes.setUploaderInfo(new URL("http://www.example.com/users/grillymcgrillerson"));
         expectedVideoAttributes.setLive(false);
+        VideoAttributes attr = (VideoAttributes) siter.next().getAttributesForExtension(Extension.VIDEO)[0];
+        assertNotNull(attr);
+        assertEquals(expectedVideoAttributes, attr);
 
-        for (SiteMapURL su : sm.getSiteMapUrls()) {
-            assertNotNull(su.getAttributesForExtension(Extension.VIDEO));
-            VideoAttributes attr = (VideoAttributes) su.getAttributesForExtension(Extension.VIDEO)[0];
-            assertEquals(expectedVideoAttributes, attr);
-        }
+        // locale-specific number format in <video:price>, test #220
+        expectedVideoAttributes = new VideoAttributes(new URL("http://www.example.com/thumbs/123-2.jpg"), "Grilling steaks for summer, episode 2",
+                        "Alkis shows you how to get perfectly done steaks every time", new URL("http://www.example.com/video123-2.flv"), null);
+        expectedVideoAttributes.setPrices(new VideoAttributes.VideoPrice[] { new VideoAttributes.VideoPrice("EUR", null, VideoAttributes.VideoPriceType.own) });
```

Comment:
Maybe a comment about why we're setting the price to null (current expected behavior is to not handle non-US locale currency values, but set them to null vs. failing).